app/src/main/java/io/branch/branchlinksimulator/MainActivity.kt

```diff
@@ -170,110 +172,97 @@ fun MainContent(navController: NavController) {
     customerEventAlias = savedAlias
     var aliasValue by remember { mutableStateOf(savedAlias) }
 
-    Column(modifier = Modifier.padding(16.dp)) {
-        Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.fillMaxWidth()) {
-            Image(
-                painter = painterResource(id = R.drawable.branch_badge_all_white),
-                contentDescription = "App Logo",
-                modifier = Modifier.size(36.dp),
-                colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.onBackground)
-            )
-            Spacer(modifier = Modifier.width(8.dp))
-            Text("Branch Link Simulator", style = MaterialTheme.typography.headlineMedium, color = MaterialTheme.colorScheme.onBackground)
+    LazyColumn(modifier = Modifier.padding(16.dp)) {
```

Comment:
really not a big fan of this lazycolumn and how i had to surround everything with `item {`